script/hassfest/docker.py

```diff
@@ -112,7 +112,7 @@
 LABEL "com.github.actions.color"="gray-dark"
 """
 
-_GO2RTC_VERSION = "1.9.6"
+_GO2RTC_VERSION = "1.9.7"
```

Comment:
If we update the min. version it would be a breaking change and not allowed in the patch release.
Also, the code still works with go2rtc. Only changing the log level will not work.

Reply:
> If we update the min. version it would be a breaking change and not allowed in the patch release.

I'm not sure I agree requiring users who decide to manage the go2rtc server by themselves to upgrade to a good version of go2rtc can be considered a breaking change.